docs/src/testing-library-js.md

```diff
@@ -18,7 +18,7 @@ If you use DOM Testing Library in the browser (for example, you bundle end-to-en
 | [screen](https://testing-library.com/docs/queries/about#screen) | [page](./api/class-page) and [component](./api/class-locator) |
 | [queries](https://testing-library.com/docs/queries/about) | [locators](./locators) |
 | [async helpers](https://testing-library.com/docs/dom-testing-library/api-async) | [assertions](./test-assertions) |
-| [user events](https://testing-library.com/docs/user-event/intro) | [actions](./api/class-locator) |
+| [user events](https://testing-library.com/docs/user-event/intro) | [actions](./input) |
```

Comment:
Looks like class-locator is the right place to point to

Reply:
Since the locators page doesn't contain an explanation about actions, I was under the impression that the actions page is better:
This is the locators:
<img width="1003" alt="image" src="https://github.com/microsoft/playwright/assets/12711091/59bedb96-daf0-4659-af14-41f18ddc600a">

And this is the actions:
<img width="1000" alt="image" src="https://github.com/microsoft/playwright/assets/12711091/e2ba7416-4caa-4145-96e0-914bc4875cf8">

But, this is only my understanding. I'll close this one and feel free to re-open if you think my comment is worth it :)
Thanks!